demo-app/src/main/AndroidManifest.xml

```diff
@@ -83,25 +83,14 @@
             <intent-filter
                 android:autoVerify="true"
                 android:label="@string/filter_call_link">
-
                 <action android:name="android.intent.action.VIEW" />
-
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-
-                <data android:scheme="https" />
-                <data android:host="@string/deeplinking_host" />
-                <data android:pathPrefix="@string/deeplinking_path_prefix" />
-            </intent-filter>
-            <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
-
-                <category android:name="android.intent.category.DEFAULT" />
-                <category android:name="android.intent.category.BROWSABLE" />
-
                 <data android:scheme="https" />
-                <data android:host="@string/deeplinking_host_legacy" />
-                <data android:pathPrefix="@string/deeplinking_path_prefix" />
+                <data android:host="getstream.io" />
+                <data android:host="pronto.getstream.io" />
+                <data android:pathPrefix="/join" />
+                <data android:pathPrefix="/video/demos" />
```

Comment:
We will probably also need to add `pathPattern` or `pathAdvancedPattern` to make sure we only match URLs that contain the ID. 
Otherwise there is a bug - if you enter https://getstream.io/video/ on mobile browser and then click `Explore demo` it will open `https://getstream.io/video/demos` - which will then open our app and but without a call ID.

Reply:
Or maybe we just need to change this:
`<data android:pathPrefix="/video/demos" />`
to this?
`<data android:pathPrefix="/video/demos/join" />`